### ISSUE TITLE
Make DeterminantalPointProcesses.jl compatible with Distributions.jl?

### ISSUE THREAD

**Issue by gajomi** [alshedivat/DeterminantalPointProcesses.jl#3]
Is there any reason that the various DPP types are not subtypes of Multivariate Distribution? This could open open compatibility with the wider julia ecosystem. I might be willing to do the work to make this so if it was desirable.

**Comment by alshedivat** [comment]
Good point. There is no particular reason -- integration with a wider ecosystem is something I didn't consider back when implemented DPP. It'd be very nice to have DPP compatible with `Distributions.jl`.
